Projects/Feature/HomeFeature/Sources/Scene/View/MealContentStackView.swift

```diff
@@ -20,9 +20,14 @@ final class MealContentStackView: UIStackView {
 
     public func updateContent(meals: [String]) {
         self.removeAllChildren()
-
-        guard !meals.isEmpty else { return }
-        let mealViews = meals.map { DotoriLabel($0) }
+        var mealViews: [DotoriLabel] = []
+        if meals.isEmpty {
+            let emptyLabels = (1...6).map { _ in DotoriLabel("") }
+            mealViews.append(DotoriLabel("급식이 없습니다."))
+            mealViews.append(contentsOf: emptyLabels)
+        } else {
+            mealViews = meals.map { DotoriLabel($0) }
+        }
```

Comment:
```suggestion
        let mealViews: [DotoriLabel] = if meals.isEmpty {
            (1...6).map { _ in DotoriLabel("") }
        } else {
            meals.map { DotoriLabel($0) }
        }
```
이게 Swift 5.9 문법인데 Github Action에서 Swift 버전이 5.9인지는 모르겠긴하겠네요
아무튼 이 새 문법 어떤가요